scripts/update_dependencies.sh

```diff
@@ -0,0 +1,40 @@
+# Pip dependency update script
+# Usage:
+# bash <path to script> [Optional branch name]
+#
+# If the branch name is specified install the requirements from that branch
+# otherwise install from develop
+set -e
+
+if [ "$1" ]
+  then
+    REQ_BRANCH="$1"
+else
+    REQ_BRANCH="develop"
+fi
+
+# make sure bluemira is in the current environment
+BM_INSTALLED=$(pip list | grep bluemira | awk 'NR==1 {print $1}')
+[ "$BM_INSTALLED" = "bluemira" ] || { echo "bluemira not found in the current python environment, exiting" && exit 1; }
+
+readonly BLUEMIRA_ROOT="$(realpath "$(dirname "$0")"/..)"
+echo 'Bluemira directory is '$BLUEMIRA_ROOT
```

Comment:
I would double quote all these variable accesses to guard against word splitting.

```suggestion
echo "Bluemira directory is $BLUEMIRA_ROOT"
```